README.md

```diff
@@ -1,36 +1,48 @@
 
 Originally developed by [Carlos Tadeu Panato Junior](https://github.com/cpanato/).
 
+![image](https://user-images.githubusercontent.com/13119842/58750029-df501000-845a-11e9-88f2-63fc0db5bc26.png)
+
 ## Configuration
 
-### Mattermost side:
+### Step 1: Configure plugin in Mattermost
+
+1. Go to **System Console > Plugins > AWS SNS**:
+
+  1. Set the channel to send notifications to, specified in the format `teamname,channelname`. If the specified channel does not exist, the plugin will create the channel for you. 
+      - Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel
+
+  2. Set authorized users who can accept AWS SNS subscriptions. Must be a comma-separated list of user ID's.
+      - Tip: Use the [mattermost user search](https://mattermost.com/pl/cli-mattermost-user-search) CLI command to determine a user ID.
+
+  3. Set the username that this integration is attached to.
+  4. Generate a token used for an AWS SNS subscription. Copy this value as you will use it in a later step.
+
+2. Go to **System Console > Plugins > Management** and click **Enable** to enable the AWS SNS plugin.
+
+Note: If you are running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-aws-SNS/releases), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+
+### Step 2: Configure plugin in Amazon AWS
+
+1. Create an [AWS CloudWatch alarm for your instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-cloudwatch-createalarm.html). 
+
+2. Create an AWS SNS Topic with an HTTPS subscription to https://your-mattermost-url/plugins/com.mattermost.aws-sns?token=your-mattermost-token`, where `your-mattermost-url` refers to your Mattermost URL, and `your-mattermost-token` was generated on a previous step. [Follow this documentation](https://docs.safe.com/fme/html/FME_Server_Documentation/ReferenceManual/Amazon_SNS_Publisher_Configure_AWS_Subscription.html) for additional configuration options.
 
-  - Install the plugin in your Mattermost instance
-  - Configure the plugin
-    - Set the `Team` and `Channel` that you want to receive the messages. If the Channel does not exist the plugin will create the channel for you.
-    Set the `Team` and `Channel` with comma, ie. `teamb,channelx`.
-  - Set the `AllowedUserIds` with the users ids, this will allow specific users to approve the subscription from AWS SNS.
-  - Set the `Username`, this will be user that will post the messages.
-  - Set the `Token`, this will be use when creating the SNS subscription.
+3. Switch to the Mattermost channel you configured to receive notifications. Accept the subscription posted to the channel.
```

Comment:
I'm not sure if this step is accurately described. I didn't (yet at least) have time to set up a CloudWatch account and test this user experience.

Reply:
Me neither, sorry I can't be of more help

Reply:
When you configure the AWS SNS to send the messages to MM, AWS SNS will send a message to confirm the subscription, this message will arrive in the channel you selected to receive the messages and then you need to confirm the subscription clicking in the action button

Reply:
Thank you! Updated the note accordingly